src/typeahead/index.tsx

```diff
@@ -53,6 +55,8 @@ export interface TypeaheadProperties {
 	itemDisabled?: ListProperties['disabled'];
 	/** Flag to indicate if values other than those in the resource can be entered, defaults to true */
 	strict?: boolean;
+	/** Flag to indicate if drop down arrow should be shown in trailing section of text input, defaults to false */
+	showDropDownButton?: boolean;
```

Comment:
feels like a bit of a mouthful! I don't have a better idea other than we could potentially drop `show`

Reply:
I prefer dijit's `hasDownArrow` if that helps.

Reply:
@matt-gadd That does help! Thanks, I'm sold.